lib/engines/content_block_manager/app/assets/stylesheets/content_block_manager/application.scss

```diff
@@ -1,3 +1,13 @@
 @import "components/filter-options-component";
 @import "components/host-editions-table-component";
 @import "components/timeline-component";
+
+.content-block-manager-header {
```

Comment:
not sure if it's a faux pas to put this direct into `application.scss` ? but it also doesn't seem heavyweight enough to have its own component.